pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -893,9 +893,17 @@ func (r *ReconcileConstraintTemplate) updateTemplateWithBlockVAPBGenerationAnnot
 	if ct.Annotations != nil && ct.Annotations[constraint.VAPBGenerationAnnotation] == constraint.VAPBGenerationUnblocked {
 		return noRequeue, nil
 	}
+	latestCT := &v1beta1.ConstraintTemplate{}
+	err := r.Get(ctx, types.NamespacedName{Name: ct.GetName()}, latestCT)
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			return time.Duration(time.Millisecond), err
```

Comment:
Figuring out what is the default in kk for retries if any, otherwise we need to figure out default here.